builds/Makefile

```diff
@@ -328,8 +326,8 @@ $(MAC_DIRECTORIES) $(BUILD_DESTDIR):
 mac-facade-check: $(MAC_BCL_TARGETS)
```

Comment:
We'll need to get rid of the `FACADE_SUBDIRS_MK` which is included directly from mono sources in https://github.com/xamarin/xamarin-macios/blob/03669d4c030b2313d2037493307e3cb2d4f3bd88/builds/Makefile#L188

There's two options: 1) just include all *.dll in the ios/mac-bcl folders and trust mono to not put unwanted files there 2) hardcode list of assemblies/facades here. Any preferences?

Reply:
I think the [Xamarin.Linker.SdkTest](https://github.com/xamarin/xamarin-macios/blob/9945186b9e22334e044e6cb4537c3031b710a402/tests/mtouch/SdkTest.cs#L32) should cover this scenario, so I think we can just remove the whole facade check from this makefile.

@spouliot what do you think?

Reply:
Actually the facade check is just one part of the problem, we'd still need a list of the facade assemblies. Solution 1) won't work when building from source, so I think we need a dual solution: keep the existing code when building from source, and then use solution 1) when we're using the mono archive.

We still have a hard-coded list of facade assemblies elsewhere in our code (in tests), so we'll be notified if new facade assemblies are added to the archive (the corresponding test will start to fail).